examples/dynamics/absolutely inelastic collision.py

```diff
@@ -0,0 +1,46 @@
+#inelastic collision transforms two objects with masses m1, m2 and velocities v1 and v2 to one object with mass m = m1+m2 and unknown velocity to be resolved
+#assert 1-dimentional env
+
+
+#!/usr/bin/env python3
+from symplyphysics import (
+    units, convert_to, SI
+)
+from symplyphysics.definitions   import  momentum_is_mass_times_velocity as momentum_defined
+from symplyphysics.laws.dynamics import  momentum_after_collision_equals_to_momentum_before as momentum_law
+
+m1 = units.Quantity('m1')
+v1 = units.Quantity('v1')
+m2 = units.Quantity('m2')
+v2 = units.Quantity('v2')
+m = units.Quantity('m3')
+
+SI.set_quantity_dimension(m1, units.mass)
+SI.set_quantity_dimension(m2, units.mass)
+SI.set_quantity_dimension(m, units.mass)
+
+SI.set_quantity_dimension(v1, units.velocity)
+SI.set_quantity_dimension(v2, units.velocity)
+
+SI.set_quantity_scale_factor(m1, 0.01 * units.kilogram)
+SI.set_quantity_scale_factor(m2, 75 * units.kilogram)
+SI.set_quantity_scale_factor(m, m1 + m2 * units.kilogram)
+
+SI.set_quantity_scale_factor(v1, 700 * units.meter / units.second)
+SI.set_quantity_scale_factor(v2, 0 * units.meter / units.second)
```

Comment:
Add some high level description here, that two bodies with such velocities and masses collide, and what happens next.